### ISSUE TITLE
[BUG] Create ILLA Builder error: Docker responded with status code 400: invalid mount

### ISSUE THREAD

**Issue by Lan-Phan** [illa-family/illa#57]
**Describe the bug**
<img width="648" alt="Screen Shot 2023-01-11 at 1 40 26 PM" src="https://user-images.githubusercontent.com/106818620/211735933-0f890b7d-999f-4ac3-8907-abccc3323953.png">

**To Reproduce**
1. Install Rust via Homebrew
2. Install illa cli tool via cargo: cargo install illa
3. Run command: illa deploy --self
4. See an error above

**Expected behavior**
ILLA Builder is deployed ok

**Screenshots**

**Desktop (please complete the following information):**
 - Macbook Pro, M1
 - OS: Mac OS 12.4
 - Browser: Chrome
 - Version illa cli: 1.26

**Additional context**
 - Rust 1.66
 - Run Docker via Colima
 - Docker version 20.10.16
 - Colima version 0.5.2

**Comment by karminski** [comment]
@Lan-Phan Hi, the Colima with Docker is an untested environment for illa and we haven't tried it. Please try Docker Desktop. 

And we found the issue (https://github.com/abiosoft/colima/issues/503), Does the Colima have the mount issue?

**Comment by Lan-Phan** [comment]
Hi @karminski ,

Thanks for your info. Let I dig more into Colima ...
In the meantime, would you please show me the path which is used in mount volume when running illa deploy command?

Thanks.

**Comment by Lan-Phan** [comment]
I confirm this issue is from Colima.

Here are steps I did to make it work:
+ upgrade MacOS to version 13.1 (Ventura)
+ upgrade colima/lima to the latest version
+ remove all colima / lima old data
+ using vm type vz and virtiofs mount type and mount /tmp/illa-data writable for VM lima
+ run docker run from script here https://github.com/illacloud/deploy-illa-manually/blob/main/docker/scripts/run.sh , just 1 minor change is to mount /var/lib/postgresql/data in container to sub-directory in /tmp/illa-data (for ex: /tmp/illa-data/pg) (see my comment here https://github.com/abiosoft/colima/issues/54#issuecomment-1400796076 )

It's better if I can change /tmp/illa-data to other path in HOME folder, do you have plan to support it?

**Comment by karminski** [comment]
@Lan-Phan Great works! Your posts help a lot. 

We can make the mount location configurable. I will reply to this issue once it is completed.

**Comment by naj1n** [comment]
@Lan-Phan thanks for your issue. It will lead to some active work on the configurable mount path.

**Comment by naj1n** [comment]
@Lan-Phan Now, you can make the mount location configurable with `--mount` flag of subcommand `deploy`. It's available in the new version v1.2.8.

**Comment by Lan-Phan** [comment]
* upgrade illa to v1.2.8
* Run:
```
colima start --mount-type virtiofs --vm-type=vz
illa deploy --self --mount ~/illa-data
```

It works like a charm now, thank @naj1n ,  close this.
